packages/blockchain/package.json

```diff
@@ -41,7 +41,7 @@
     "ethashjs": "~0.0.7",
     "ethereumjs-block": "~3.0.0",
     "ethereumjs-common": "^1.5.0",
-    "ethereumjs-util": "~6.1.0",
+    "ethereumjs-util": "^7.0.2",
```

Comment:
I have reverted it back to a caret range, to comply with the other monorepo packages. 
Looking at old versions of this line led to this PR https://github.com/ethereumjs/ethereumjs-blockchain/pull/33, proving this was a protection mechanism for a hardfork changes. 

Any opinions here, @holgerd77?

Reply:
This is from some really old ages 😄 , no reason for special rules here any more.

Think its very beneficial if we end up using caret ranges at least for our own internal dependencies all over the place and just follow semver without exception. This was quite different some time (2-3 years) ago but we have gotten quite some way here and I think we are now with most (all?) of the (actively worked on) libraries in a semver-compliant release process.